packages/quantic/force-app/main/default/lwc/exampleSearch/exampleSearch.html

```diff
@@ -17,7 +17,7 @@
                         <c-quantic-sort class="slds-order_1 slds-medium-order_2 slds-size_1-of-1 slds-medium-size_3-of-12" engine-id={engineId}>
                         </c-quantic-sort>
                     </div>
-
+                    <c-quantic-breadcrumb engine-id={engineId}></c-quantic-breadcrumb>
```

Comment:
I would also recommend adjusting the name, since `breadcrumb` suggests a single entity, but this component can display multiple instances.

Reply:
quantic-breadcrumbs ? 😂
Why the "manager" part exactly? I don't really get it, it doesn't manage breadcrumbs, to me a "manager" implies there is another part somewhere that are the actual breadcrumbs being managed?

(I'll totally change it, I don't really mind, I was just curious)

Reply:
Using the plural could be an option. I typically lean towards using the singular for component names, like `quantic-breadcrumb-container` or `...manager` since it provides clear all and collapse functionality; it's more than just the list of breadcrumbs.

I can see that level of accuracy being unnecessary though, so I leave the call to the SFINT team. At the end of the day, the most important thing is that devs understand the name,

Reply:
Even if `breadcrumb-container` might be a better name for this component, I think we should remain as coherent with atomic as possible, otherwise it will become quite counter-intuitive to developers using both atomic and quantic.

I would recommend to go with `quantic-breadcrumb-manager`.

Reply:
I agree. IMO staying consistent with atomic is the way to go.

Reply:
done